src/actions/transformMatch.js

```diff
@@ -211,6 +211,16 @@ function transformMatch(m) {
     }
     newPlayer.buybacks = (player.buyback_log || []).length;
     newPlayer.total_gold = (player.gold_per_min * m.duration) / 60;
+    if (m.game_mode === 18) {
+      const arr = [];
+      arr[0] = 1;
```

Comment:
what is 1?

Reply:
The table doesnt show the index[0] of the array, so i just pushed a dummy value